addons/purchase_requisition/models/purchase.py

```diff
@@ -91,9 +91,9 @@ def _onchange_requisition_id(self):
             taxes_ids = fpos.map_tax(line.product_id.supplier_taxes_id.filtered(lambda tax: tax.company_id == requisition.company_id)).ids
 
             # Compute quantity and price_unit
-            if line.product_uom_id != line.product_id.uom_po_id:
-                product_qty = line.product_uom_id._compute_quantity(line.product_qty, line.product_id.uom_po_id)
-                price_unit = line.product_uom_id._compute_price(line.price_unit, line.product_id.uom_po_id)
+            if line.product_uom_id != line.product_id.uom_id:
+                product_qty = line.product_uom_id._compute_quantity(line.product_qty, line.product_id.uom_id)
+                price_unit = line.product_uom_id._compute_price(line.price_unit, line.product_id.uom_id)
```

Comment:
Should be checked with PO if we still need to do this conversion now that sellers can have different UoMs from the purchase UoM (i.e. there's no reason why we can't do the same here.)

Example case, create a blanket order with a different UoM from the `product_id.uom_id` => create a PO. Current behavior forces it to be in the purchase UoM, but I don't know a reason for leaving it restricted (i.e. it might be old code/behavior).

Reply:
@ticodoo yes I agree with you, no need to convert